content/en/02-installing-pixie/03-install-guides/01-community-cloud-for-pixie.md

```diff
@@ -76,6 +76,13 @@ px-operator         vizier-operator
 
 To deploy Pixie to another cluster, change your `kubectl config current-context` to point to that cluster. Then repeat the same deploy commands shown in this step.
 
+If the `px deploy` does not successfully complete you might retry it after deleting all the nodes and the namespace as sometimes the deploy nodes can get into a bad state:
```

Comment:
@nserrino said this issue would be fixed in the next release [here](https://github.com/pixie-io/pixie/pull/324#issuecomment-931614812) so I think we should just leave this out of the docs.

Reply:
I've removed it, thanks!